src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,151 @@
+package duke;
+
+import duke.commands.ByeCommand;
+import duke.commands.Command;
+import duke.commands.DeadlineCommand;
```

Comment:
I'm not sure you require 1 class for every single command. Perhaps making these comments the functions of classes such as Task, Todo, Deadline, Event etc?